doc/source/quick_start.md

```diff
@@ -14,253 +14,81 @@ kernelspec:
-
-A {func}`~geoutils.Raster.reproject` involves a change in {attr}`~geoutils.Raster.crs` or {attr}`~geoutils.Raster.transform`, while a {func}`~geoutils.Raster.crop` only involves a change
-in {attr}`~geoutils.Raster.bounds`. Using {func}`~geoutils.Raster.polygonize` allows to generate a {class}`~geoutils.Vector` from a {class}`~geoutils.Raster`,
-and the other way around for {func}`~geoutils.Vector.rasterize`.
-
-```{list-table}
-   :widths: 30 30 30
-   :header-rows: 1
-
-   * - **Class**
-     - {class}`~geoutils.Raster`
-     - {class}`~geoutils.Vector`
-   * - Warping/Reprojection
-     - {func}`~geoutils.Raster.reproject`
-     - {func}`~geoutils.Vector.reproject`
-   * - Cropping/Clipping
-     - {func}`~geoutils.Raster.crop`
-     - {func}`~geoutils.Vector.crop`
-   * - Rasterize/Polygonize
-     - {func}`~geoutils.Raster.polygonize`
-     - {func}`~geoutils.Vector.rasterize`
-```
-
-All methods can also be passed any number of georeferencing arguments such as {attr}`~geoutils.Raster.shape` or {attr}`~geoutils.Raster.res`, and will
-naturally deduce others from the input {class}`~geoutils.Raster` or {class}`~geoutils.Vector`, much as in [GDAL](https://gdal.org/)'s command line.
-
-
-## Higher-level analysis tools
-
-GeoUtils also implements higher-level geospatial analysis tools for both {class}`Rasters<geoutils.Raster>` and {class}`Vectors<geoutils.Vector>`. For
-example, one can compute the distance to a {class}`~geoutils.Vector` geometry, or to target pixels of a {class}`~geoutils.Raster`, using
-{func}`~geoutils.Vector.proximity`.
-
-As with the geospatial handling functions previously listed, many analysis functions can take a {class}`~geoutils.Raster` or {class}`~geoutils.Vector` as a
-**reference to utilize** during the operation. In the case of {func}`~geoutils.Vector.proximity`, passing a {class}`~geoutils.Raster` serves as a reference
-for the georeferenced grid on which to compute the distances.
-
-```{code-cell} ipython3
-# Compute proximity to vector on raster's grid
-rast_proximity_to_vec = vect.proximity(rast)
-```
-
-```{note}
-Right now, the array {attr}`~geoutils.Raster.data` of `rast` is still not loaded. Applying {func}`~geoutils.Raster.crop` does not yet require loading,
-and `rast`'s metadata is sufficient to provide a georeferenced grid for {func}`~geoutils.Vector.proximity`. The array will only be loaded when necessary.
-```
 
-## Quick plotting
+# Buffer the vector by 500 meters no matter its current projection system
+vect_buff = vect.buffer_metric(500)
 
-To facilitate the analysis process, GeoUtils includes quick plotting tools that support multiple colorbars and implicitly add layers to the current axis.
-Those are build on top of {func}`rasterio.plot.show` and {func}`geopandas.GeoDataFrame.plot`, and relay any argument passed.
+# Create boolean mask of vector on same grid/CRS as raster using it as "match-reference"
```

Comment:
'boolean' can be removed here without loss of clarity

Reply:
Done!